plugins.json

```diff
@@ -733,13 +733,13 @@
     },
     "signal": {
         "title": "Signal",
-        "version": "1.0.1",
+        "version": "1.0.2",
         "author": "Benedikt Hopmann",
         "license": "MIT",
         "description": "Receive Kanboard notifications on Signal (via signal-cli).",
-        "homepage": "https://github.com/stratmaster/kanboard-plugin-signal",
-        "readme": "https://raw.githubusercontent.com/stratmaster/kanboard-plugin-signal/master/README.md",
-        "download": "https://github.com/stratmaster/kanboard-plugin-signal/releases/download/v1.0.1/Signal-1.0.1.zip",
+        "homepage": "https://github.com/bhopmann/kanboard-plugin-signal",
+        "readme": "https://github.com/bhopmann/kanboard-plugin-signal/blob/master/README.md",
+        "download": "https://github.com/bhopmann/kanboard-plugin-signal/archive/refs/tags/v1.0.2.zip",
```

Comment:
The download URL should be `https://github.com/bhopmann/kanboard-plugin-signal/releases/download/v1.0.2/Signal-1.0.2.zip`